en/FeedHeader.md

```diff
@@ -0,0 +1,10 @@
+## Practice Recommendations Organized by Message
+
+### FeedHeader
+
+| Field Name | Recommendation |
+| --- | --- |
+| gtfs_realtime_version | Current version is "2.0".  All GTFS-realtime feeds should be "2.0" or higher, as early version of GTFS-realtime did not require all fields needed to represent various transit situations adequately. |
+| incrementality | | 
+| timestamp | This timestamp should be greater than all timestamps for individual entities ([TripUpdate](#TripUpdate) and [VehiclePosition](#VehiclePosition)), and should not decrease between two sequential iterations. |
+|  | The GTFS-realtime header `timestamp` value should always change if the feed contents change - the feed contents must not change without updating the header `timestamp`.<br>*Common mistakes* - If there are multiple instances of GTFS-realtime feed behind a load balancer, each instance may be pulling information from the realtime data source and publishing it to consumers slightly out of sync. If a GTFS-realtime consumer makes two back-to-back requests, and each request is served by a different GTFS-realtime feed instance, the same feed contents could potentially be returned to the consumer with different timestamps.<br>*Possible solution* - Configure the load balancer for "sticky routes", so that the consumer always receives the GTFS-realtime feed contents from the same GTFS-realtime instance. |

```

Comment:
Usage of "must" indicates this belongs in the spec.

Reply:
Addressed in c7d22d9a4e6857f84c3e764fc1b368dccce1e458 and 6cc1687f65aa831859e6c5fe430e787d161b2b8e